src/components/legend/color.tsx

```diff
@@ -49,16 +49,17 @@ export const Color = memo(
     });
 
     return (
-      <div
+      <button
+        type="button"
         onClick={hasColorPicker ? onClick : undefined}
         className={colorClasses}
-        aria-label="series color"
+        aria-label={hasColorPicker ? 'change series color' : 'series color'}
```

Comment:
We should also mention what the current color and buttons are assumed to be interactive so I think we can tweak this to something like:
```suggestion
        aria-label={`series color: ${colorName}`}
```

But putting this together with my comment above, we cannot leave an `aria-label` on a `div`. In that case, we should add the label to the `<svg>` of the icon.

Reply:
Yeah I think we just split this and have the actionable element say ```aria-label={`series color: ${colorName}`}``` and the actionable say ```aria-label={`change series color: ${colorName}`}```